schema/deploy/functions/update-modified-timestamp.sql

```diff
@@ -0,0 +1,18 @@
+-- Deploy seattleflu/schema:functions/update-modified-timestamp to pg
+
+begin;
+
+create or replace function warehouse.update_modified_timestamp() returns trigger as $$
+
+    begin
+        NEW.modified = now();
+        return new;
+    end;
+$$ language plpgsql
+    security definer
```

Comment:
`security definer` functions should be used rarely and with explicit reason. This function doesn't need it, so I assume this is a copy-paste thing. It is important to understand all the code you're adding/using, esp. when cribbing from existing code. I'm happy to explain unknown bits of code anytime. :-)

Reply:
I remember adding `security definer` to an existing trigger function at some point, but I can't remember why it was necessary. Is this in earlier trigger functions because we changed the schema owner at some point?

Reply:
Remembering why something was done is exactly why we write good commit messages: https://github.com/seattleflu/id3c/commit/1efa888834e553efcf5374058f9b5dedb2e1dee4 :-D